src/Microsoft.DotNet.Wpf/test/System.Xaml/System.Xaml.Tests.csproj

```diff
@@ -0,0 +1,15 @@
+<Project Sdk="Microsoft.NET.Sdk">

```

Comment:
can you name this file "System.Xaml.UnitTests.csproj". i think that way it get's automatically picked up and ran by arcade.

Reply:
See https://github.com/dotnet/arcade/blob/master/Documentation/ArcadeSdk.md#project-name-conventions